docs/source/release_notes/release-notes.rst

```diff
@@ -35,41 +35,64 @@ Changelog
 develop
 =======
 
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+0.27.0
+=======
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
 
     *    - Type
          - Change
 
+    *    - |new|
+         - AtlasDB now supports stream store compression.
+           Streams can be compressed client-side by adding the ``compressStreamInClient`` option to the stream definition.
+           Reads from the stream store will transparently decompress the data.
+
+           For information on using the stream store, see :ref:`schemas-streams`.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1357>`__)
+
     *    - |improved|
-         - ``StreamStore.loadStream`` now actually streams the data, if the data does not fit in memory.
-           This means that getting the first byte of the stream now has constant-time performance, compared to being
-           linear in terms of stream length, as it was previously.
+         - ``StreamStore.loadStream`` now actually streams data if it does not fit in memory.
+           This means that getting the first byte of the stream now has constant-time performance, rather than linear in terms of stream length as it was previously.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1341>`__)
 
     *    - |improved|
-         - Oracle query performance improvement; table name cache now global to KVS level.
+         - Increased connection pool idle timeout to 10 minutes, and reduced eviction check frequency to 20-30 seconds at 1/10 of connections.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1336>`__)
+
+    *    - |new|
+         - There is a new configuration called ``maxConnectionBurstSize``, which configures how large the pool is able to grow when receiving a large burst of requests.
+           Previously this was hard-coded to 5x the ``poolSize`` (which is now the default for the parameter).
+
+           See :ref:`Cassandra KVS Config <cassandra-kvs-config>` for details on configuring AtlasDB with Cassandra.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1336>`__)
+
+    *    - |improved|
+         - Improved the performance of Oracle queries by making the table name cache global to the KVS level.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1325>`__)
 
     *    - |fixed|
-         - Oracle value style caching limited in scope to per-KVS, previously per-JVM,
-           which could have in extremely rare cases caused issues for users in odd configurations.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1235>`__)
+         - Oracle value style caching limited in scope to per-KVS, previously per-JVM, which could have in extremely rare cases caused issues for users in non-standard configurations.
```

Comment:
From the PR: "The other oracle-related caches have a different problem, which is that unlike the table unmapper, they are already /too/ global.
They are java static when they really should be also per-KVS instead of per-JVM, because there are legitimate cases when you can have multiple Oracle KVSs in the same JVM that don't point to the same DB. (split KVS configurations, or more likely someone doing a KVS migration to move from one Oracle DB to another)"